source/Halibut.Tests/FriendlyHtmlPageTests.cs

```diff
@@ -34,7 +34,7 @@ static DelegateServiceFactory GetDelegateServiceFactory()
         public async Task SupportsHttpsGet(string uriFormat)
         {
             var services = GetDelegateServiceFactory();
-            using (var octopus = new HalibutRuntime(services, Certificates.Octopus))
+            using (var octopus = new HalibutRuntimeBuilder().WithServiceFactory(services).WithServerCertificate(Certificates.Octopus).WithAsyncHalibutFeatureEnabled().Build())
```

Comment:
Do these tests now only test Async and so now were missing sync coverage?

Reply:
🤦🏻‍♂️ 

I've created some new test attributes to deal with sync/async when using `TestCase`, so we can define multiple `TestCase` attributes on a tests which will then get run for both sync and async.